packages/amplify-category-storage/provider-utils/awscloudformation/index.js

```diff
@@ -41,7 +41,7 @@ function migrateResource(context, projectPath, service, resourceName) {
     return;
   }
 
-  return migrate(context, projectPath, resourceName);
+  return migrate(projectPath, resourceName);
```

Comment:
The `migrateResource` method is used for both `S3` and `DynamoDB`

https://github.com/aws-amplify/amplify-cli/blob/92895ca8bd9a37ca96b7aeb0ccfda8a379aa8e43/packages/amplify-category-storage/provider-utils/awscloudformation/service-walkthroughs/s3-walkthrough.js#L290-L294


https://github.com/aws-amplify/amplify-cli/blob/92895ca8bd9a37ca96b7aeb0ccfda8a379aa8e43/packages/amplify-category-storage/provider-utils/awscloudformation/service-walkthroughs/dynamoDb-walkthrough.js#L392-L394.

The right fix will be to add `context` to DDB service walkthrough.

Reply:
ahh yeah I was wondering which was the right way to go about it. Will do so right away.